src/DuneFormatter.ml

```diff
@@ -0,0 +1,24 @@
+open Import
+
+let format (document : TextDocument.t) options token =
+  let lastLine = document.lineCount - 1 in
+  let lastCharacter =
+    String.length (TextDocument.lineAt document lastLine).text
+  in
+  (* selects entire document range *)
+  let fullDocumentRange =
+    Range.make ~startLine:0 ~startCharacter:0 ~endLine:lastLine
+      ~endCharacter:lastCharacter
+  in
+  (* text of entire document *)
+  let documentText = TextDocument.getText document fullDocumentRange in
+
+  ChildProcess.exec "dune format-dune-file" ~stdin:documentText
```

Comment:
Don't we have the `Cmd` module for this?

Reply:
Ahh. Must have overlooked it sorry.